teste.py

```
@@ -0,0 +1,12 @@
+import numpy as np
```

Ruff (0.8.0)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)